docs/gettingstarted/buildingtest.rst

```diff
@@ -384,6 +375,18 @@ where test failed to run since we provided invalid executor.
     .. command-output:: buildtest build -b tutorials/invalid_executor.yml
         :returncode: 1
 
+Troubleshoot Tests (``buildtest build --dry-run``)
```

Comment:
Just rename this title to `Dry Run` because we do have a section on troubleshooting https://buildtest.readthedocs.io/en/devel/troubleshooting.html

Reply:
👍